new/server-http-head.md

```diff
@@ -0,0 +1,48 @@
+---
+RFC: unassigned
+Title: Server HTTP HEAD
+Author: Jeremy Miller <jmiller@chef.io>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Server HTTP HEAD
+There is a greater resource cost than is necessary when querying the Server API named object endpoints for the existence of a single object.
+Currently, if checking for an object's existence, only HTTP `GET` requests are supported by the server. This means the entire object is fetched, consuming
+resources across the server, network and client. When viewed from a large scale perspective, this overhead can cause slow downs that can have
+compounding effects.
+
+## Motivation
+
+    As a Chef developer,
+    I want to be able to write code that queries the Server for the existence of a single object via a light-weight API call and response,
+    so that my applications can run as efficiently and as quickly as possible.
+
+## Specification
+
+The HEAD method shall be identical to GET except that the server must not return a message-body in the response.
+
+The meta-information contained in the HTTP headers in response to a HEAD request should be identical to the
+information sent in response to a GET request.
+
+The HEAD HTTP verb will be added to oc_erchef and chef-zero named object endpoints such that a client http HEAD request for
+an object name will result in a http 200 response code if it exists, 404 if it does not and 401 if the requestor does not
```

Comment:
401 should be the returned code if the request signature is invalid, 403 if the requestor does not have READ on the object.